frontend/bot/src/App.re

```diff
@@ -0,0 +1,34 @@
+open Discord;
+
+let client = Client.createClient();
+
+let handleMessageHelper = msg =>
+  switch (Array.to_list(Js.String.split(",", Message.content(msg)))) {
+  | ["$tiny"] => Message.reply(msg, "You summoned me?")
+  | ["$help"] => Message.reply(msg, Messages.help)
+  | ["$request", pk] => Faucet.handleMessage(msg, pk)
+  | ["$request", ..._] => Message.reply(msg, Messages.requestError)
+  | _ => ()
+  };
+
+let handleMessage = msg =>
+  switch (TextChannel.fromChannel(Message.channel(msg))) {
+  | None => ()
+  | Some(channel) =>
+    if (!(
```

Comment:
I think this handles the message if user is a bot and channel is not in the list?

should it be: `if (!bot && correctChannel)` instead?

Reply:
good call

Reply:
no worries :D